data/report_templates/project_summary.md

```diff
@@ -71,48 +76,63 @@ This means that our services are subject to highly stringent quality control pro
 so that you can be sure that your data is of excellent quality.
 
 Library preparation
-:   [cross] Not Swedac Accredited
+:   {{ project.accredit.library_preparation }}
 
 Sequencing data
-:   [tick] Swedac Accredited
+:   {{ project.accredit.sequencing }}
 
-Data flow
-:   [tick] Swedac Accredited
+Data Processing
+:   {{ project.accredit.data_processing }}
 
-Data processing
-:   [tick] Swedac Accredited
+Data Analysis
+:   {{ project.accredit.data_analysis }}
 
```

Comment:
Yup, that works as well (though maybe `project.show_samples_in_report` :wink:). Flexibility to choose can be added if/when we move to celery or whatever.